.github/workflows/ci.yaml

```diff
@@ -1,10 +1,11 @@
-name: GH Pages CI
+name: GH Pages CI V2
```

Comment:
Why change the name of the Pipeline? Would just leave it as is (or give it a more descriptive name)

Reply:
Fair enough. There already was a .yaml file, i just replaced its content

Reply:
Name is now "GH Pages CI for live env"